lib/bin/repl.js

```diff
@@ -7,18 +7,22 @@
 // including this file, may be copied, modified, propagated, or distributed
 // except according to the terms contained in the LICENSE file.
 
-const should = require('should'); // eslint-disable-line import/no-extraneous-dependencies
-require('../../test/assertions');
-
 const _ = require('lodash'); // eslint-disable-line import/no-extraneous-dependencies
 const { sql } = require('slonik');
 
 const container = require('../util/default-container');
 
 (async () => {
-  const context = { ..._.omit(container, 'with'), container, should, sql };
-  const contextKeys = Object.keys(context).sort();
-  console.log('Available vars:', contextKeys.join(', '));
+  const context = { ..._.omit(container, 'with'), container, sql };
+  const contextKeys = Object.keys(context);
+
+  try {
+    context.should = require('should'); // eslint-disable-line import/no-extraneous-dependencies
+    contextKeys.push('should'); // manually add due to should.js prototype pollution
```

Comment:
It doesn't work, due to something weird that `should` is doing:

```
> o = {}; o.should = require('should'); o.definitelyHas = require('should'); [ Object.keys(o), o.hasOwnProperty('should'), !!o.should, o.hasOwnProperty('definitelyHas'), !!o.definitelyHas ]
[ [ 'definitelyHas' ], false, true, true, true ]
```

Reply:
Ooh interesting, I see. Another idea is that you could initialize `context` with a null prototype so that it's not polluted by Should:

```js
const context = Object.assign(Object.create(null), { ..._.omit(container, 'with'), container, sql });
```

Reply:
> Another idea is that you could initialize `context` with a null prototype so that it's not polluted by Should

This looks more correct :+1:

Reply:
I was looking at it just now, and here's something else that's weird:

```
const o = {};
o.should = require('should');
should === require('should'); // true: should makes itself a global.
o.should === should; // false: should sets the should property such that it doesn't match the global. I suspect that require('should') is causing the assignment to `o.should` to just be ignored.
o.should === require('should'); // false
```

I think using a null prototype should address all this.

Reply:
I had another thought about this. I'm wondering, do we even need to add `should` to `context`? Since `should` adds itself to `global`, and we specify `useGlobal: true` below, I think `should` will be available in the REPL even if it's not added to `context`. That said, I think it'd still be useful to list `should` in the list of "Available vars". I think my suggestion comes down to something like:

```js
try {
  require('should');
  // should has added itself to global, so we don't need to add it to context.
  // We add 'should' to contextKeys so that it is displayed to the user.
  contextKeys.push('should');
  require('../../test/assertions');
}
```

`contextKeys` could be renamed to `displayList` or something if it feels uncomfortable that it differs from `Object.keys(context)`.

Just a thought!

Reply:
This looks neat :+1: